extensions/amp-video/0.1/flexible-bitrate.js

```diff
@@ -91,6 +91,9 @@ export class BitrateManager {
    * @param {!Element} video
    */
   manage(video) {
+    if (video.changedSources) {
+      return;
+    }
```

Comment:
If mediapoolEl was already managed, no need to redo this. It will call `changedSources` by itself on the `UpdateSourcesTask`

Reply:
plz add a code comment `if already managed.....`